cmd/cluster-config-operator/main.go

```diff
@@ -1,37 +1,21 @@
 package main
 
 import (
-	goflag "flag"
-	"fmt"
-	"math/rand"
 	"os"
-	"time"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 
-	utilflag "k8s.io/component-base/cli/flag"
-	"k8s.io/component-base/logs"
+	"k8s.io/component-base/cli"
 
 	"github.com/openshift/cluster-config-operator/pkg/cmd/operator"
 	"github.com/openshift/cluster-config-operator/pkg/cmd/render"
 	"github.com/openshift/cluster-config-operator/pkg/version"
 )
 
 func main() {
-	rand.Seed(time.Now().UTC().UnixNano())
-
-	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
-	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
-
-	logs.InitLogs()
-	defer logs.FlushLogs()
-
 	command := NewOperatorCommand()
-	if err := command.Execute(); err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
am I misreading or does the new library dump this along with standard log output (which might be redirected incorrectly to stderr) instead of stderr

Reply:
Hmm, as I understood it the core logic doesn't change much, except that they differentiate when it's a flag parse error versus a runtime error and use the corresponding method for output, in this [line](https://github.com/kubernetes/component-base/blob/v0.23.0/cli/run.go#L105)

Reply:
reverted back start command to simplify PR